src/app/pages/Privacy.svelte

```diff
@@ -0,0 +1,170 @@
+<script>
+  import Button from "../components/Button.svelte";
+</script>
+
+<style>
+  .privacy {
+    width: 50%;
+    margin: auto;
+    margin-top: 20px;
+  }
+
+  .privacy h1 h2 h3 h4 {
+    font-family: Cabin, sans-serif;
+  }
+
+  a {
+    text-decoration: none;
+  }
+</style>
```

Comment:
Instead of hard-coding the numbers in the document, use a [CSS counter](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Lists_and_Counters/Using_CSS_counters).

```suggestion
  article {
    counter-reset: clause;
  }
  
  article h3::before {
    counter-increment: clause;
    content: counter(clause) ". ";
  }
</style>
```